server/rooms.ts

```diff
@@ -861,8 +888,8 @@ export abstract class BasicRoom {
 			connection,
 			'|init|chat\n|title|' + this.title + '\n' + userList + '\n' + this.log.getScrollback() + this.getIntroMessage(user)
 		);
-		if (this.minorActivity) this.minorActivity.onConnect(user, connection);
-		if (this.game && this.game.onConnect) this.game.onConnect(user, connection);
+		if (this.minorActivity?.onConnect) this.minorActivity.onConnect(user, connection);
+		if (this.game?.onConnect) this.game.onConnect(user, connection);
```

Comment:
```suggestion
		this.minorActivity?.onConnect?.(user, connection);
		this.game?.onConnect?.(user, connection);
```

If you're gonna optional chain you might as well go all the way.